mmdet/models/necks/hrfpn.py

```diff
@@ -0,0 +1,89 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.utils.checkpoint import checkpoint
+from mmcv.cnn.weight_init import caffe2_xavier_init
+
+from ..utils import build_conv_layer, ConvModule
+from ..registry import NECKS
+
+
+@NECKS.register_module
+class HRFPN(nn.Module):
+    """HRFPN (High Resolution Feature Pyrmamids)
+    arXiv: https://arxiv.org/abs/1904.04514
+
+    Args:
+        in_channels (list): number of channels for each branch.
+        out_channels (int): output channels of feature pyramids.
+        pooling_type (str): pooling for generating feature pyramids
+            from {MAX, AVG}
+    """
+
+    def __init__(self,
+                 in_channels,
+                 out_channels,
+                 pooling_type='AVG',
+                 conv_cfg=None,
+                 norm_cfg=None,
+                 with_cp=False):
+        super(HRFPN, self).__init__()
+        assert isinstance(in_channels, list)
+        self.in_channels = in_channels
+        self.out_channels = out_channels
+        self.num_levels = len(in_channels)
```

Comment:
To be consistent with FPN, two attributes `num_ins` and `num_outs` are more preferable.